File: default_output/phoenix/story_lobby/story_lobby_migration.ex
defmodule FinalLocallity.Repo.Migrations.CreateStory_lobby do
  use Ecto.Migration

  def change do
    create table(:story_lobby) do
      add :story_lobby_uuid, :string
      add :host_user_uuid, :string
      add :title, :string
      add :public, :boolean

      timestamps()
    end

  end
end

File: default_output/flutter/story_lobby/story_lobby_migration.dart
class Database
{
    void execute(String s) {}
}

class Story_lobbyMigration
{
    void createDatabase(Database db)
    {
        db.execute("CREATE TABLE Story_lobby"
              +"story_lobby_uuid TEXT, "
              +"host_user_uuid TEXT, "
              +"title TEXT, "
              +"public --UNKNOWN TYPE--"
            ")"
        );
    }
}
